package-lock.json

```diff
@@ -0,0 +1,1184 @@
+{
+  "name": "good-influx",
```

Comment:
Interesting, I was not aware of this new feature.  I had to really dig for some documentation on this:
https://github.com/npm/npm/blob/5fe65673f9f3ab0c627aa30ed8d92c83e67b3713/doc/spec/package-lock.md

I tested with NPM version 3; install works just fine and tests pass.  So I don't think it will break for those of us not on npm v5 yet.  Though it looks like this file does not ship with the package -- so unclear what the benefit is?